src/scripts/quests/Quests.ts

```diff
@@ -305,7 +305,7 @@ class Quests implements Saveable {
                 if (ql) {
                     ql.state(questLine.state);
                     if (questLine.state == QuestLineState.started) {
-                        ql.resumeAt(questLine.quest, +questLine.initial || 0);
+                        ql.resumeAt(questLine.quest, questLine.initial);
                         if (ql.curQuestObject() instanceof MultipleQuestsQuest) {
                             ql.curQuestObject().quests.forEach((q, i) => {
                                 q.initial(questLine?.initial[i] ?? 0);
```

Comment:
Will probably also need to do something on this line for the multi quests, but haven't looked into those yet.

Reply:
We don't need to do anything for multiple quests, i think. Nothing has changed for them, since they where reviewed and merged.

Reply:
The problem is this only occurs when we manually start a questline in `Update.ts` because we don't know the initial value.
So it wouldn't have been tested at that time (which is the same time this issue was introduced)
AFAIK if we pass `0` to the initial function, the quest will complete if you've previously defeated the required amount, so chances are the same bug will happen on the multi quest if we manually start those too (and they are the first step).